server/views/next-steps.html

```diff
@@ -0,0 +1,123 @@
+        <li>
+          surface water extents are suitable for use in planning
+        </li>
+        <li>
+          surface water climate change scenarios may help to inform risk assessments, 
+          but the available data fall short of what is required to assess planned development 
+        </li>
+        <li>
+          surface water depth information should not be used for planning purposes
+        </li>
+      </ul>
+
+      <p class="govuk-body">
+        To find out about other factors that might affect the flood risk of this location, you should also check:
+      </p>
+
+      <ul class="govuk-list govuk-list--bullet">
+        <li>
+          <a class="govuk-link" href="https://check-long-term-flood-risk.service.gov.uk/map?easting={{floodData.centreOfPolygon.x}}&northing={{floodData.centreOfPolygon.y}}&map=SurfaceWater">
+            reservoir flood risk
+          </a>
+        </li>
+        <li>
+          groundwater flood risk - you could use the
+          <a class="govuk-link" href="https://www.bgs.ac.uk/datasets/groundwater-flooding/">
+            British Geological Survey groundwater flooding data
+          </a>,
+          <a class="govuk-link" href="https://www.gov.uk/government/collections/groundwater-current-status-and-flood-risk">
+            groundwater: current status and flood risk
+          </a>
+          and the guide on
+          <a class="govuk-link" href="https://www.gov.uk/guidance/mining-and-groundwater-constraints-for-development">
+            mining and groundwater constraints for development
+          </a>
+          - further information may be available from the lead local flood authority (LLFA) and your
+          local planning authority's strategic flood risk assessment (SFRA)
+        </li>
+        <li>
+          your local planning authority's strategic flood risk assessment (SFRA), which includes future flood risk.
+        </li>
+      </ul>
+
+      <h3 class="govuk-heading-m">
+        Other data you may need
+      </h3>
+
+      <p class="govuk-body">You will also need to assess <a
+          href="https://www.gov.uk/guidance/flood-risk-and-coastal-change#para41">residual
+          risk</a> from flood defence breach in this location.
+          If available, we will provide breach hazard mapping data to help you do this. 
+      </p>
+
+      <h3 class="govuk-heading-m">
+        Get help
+      </h3>
+
+      <p class="govuk-body">
+        For help deciding what flood risk information you need, 
+        email the Environment Agency team in {{ contactData.AreaName }} at {{ contactData.EmailAddress }}
+      </p>
+
+      <h3 class="govuk-heading-m">
+        Download a flood map for this location
+      </h3>
+
+      {% include "./partials/product-1-form.html" %}
+
+      <h3 class="govuk-heading-m">
+        Order flood risk data for rivers and the sea
+      </h3>
+
+      {% include "./partials/order-product-four.html" %}
+    </div>
```

Comment:
Can we get the naming of the 2 partials to be consistent, whichever variant

Reply:
sure - it crossed my mind too. The 2nd one isn't a form, so I was loathe to call it that. Will change the first.